core/src/main/java/com/adobe/aem/commons/assetshare/components/details/impl/MetadataImpl.java

```diff
@@ -152,7 +152,17 @@ public boolean isEmpty() {
         if (StringUtils.isBlank(getPropertyName())) {
             return true;
         } else {
-            return combinedProperties.get(getPropertyName()) == null;
+            Object val = combinedProperties.get(getPropertyName());
+            if (val == null) {
+                return true;
+            } else if (val instanceof String) {
+                return StringUtils.isBlank((String) val);
+            } else if (val instanceof String[]) {
+                String[] vals = (String[]) val;
```

Comment:
Thought of checking (Object[]) as well, checking if length is 1 and first value is null, but I'm not certain that a null value is allowed within arrays.